WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -45,5 +46,31 @@ public static string GetDataDir(string appName)
 
 			return directory;
 		}
+
+		/// <summary>
+		/// Executes a command with bash.
+		/// https://stackoverflow.com/a/47918132/2061103
+		/// </summary>
+		/// <param name="cmd"></param>
+		public static void BashExec(string cmd)
+		{
+			var escapedArgs = cmd.Replace("\"", "\\\"");
+
+			var process = new Process
+			{
+				StartInfo = new ProcessStartInfo
+				{
+					RedirectStandardOutput = true,
+					UseShellExecute = false,
+					CreateNoWindow = true,
+					WindowStyle = ProcessWindowStyle.Hidden,
+					FileName = "/bin/bash",
+					Arguments = $"-c \"{escapedArgs}\""
+				}
+			};
+
+			process.Start();
+			process.WaitForExit();
```

Comment:
IMO we should check the `ExitCode` just to be sure the command ended okay, otherwise throw and exception.